lib/webpacker/dev_server.rb

```diff
@@ -11,7 +11,7 @@ def initialize(config)
     @config = config
   end
 
-  def running?
+  def configured_and_running?
```

Comment:
By renaming this method, it's clear that there's no cost for checking if the devServer is "configured" on a production environment.

Reply:
I know I flagged this, but seeing it in context, it actually looks worse. I think we were better off with just #running?. Which is kinda ironic because I vaguely remember flipflopping on this exact question when we originally made web packer!